src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -354,7 +354,7 @@ namespace AppInstaller::Repository
         }
         else
         {
-            auto source = sourceList.GetCurrentSource(name);
+            auto source = includeHidden ? sourceList.GetSource(name) : sourceList.GetCurrentSource(name);

```

Comment:
1. I don't like the name of `GetSource()` and `GetCurrentSource()`, I had absolutely no idea what the difference was until I went to look at the definition.
2. The comments for these two functions mention tombstones. We should also add a mention of hidden by policy.
3. Would this resurrect tombstone-d sources?